tox.ini

```diff
@@ -18,33 +18,27 @@ deps = {[base]deps}
 [testenv:flake8]
 deps =
     {[base]deps}
-    mccabe==0.3.1
-    pep8==1.7.0
-    pyflakes==1.0.0
-    flake8==2.5.1
-    pep257==0.7.0
-    flake8-docstrings==0.2.4
-    flake8-quotes==0.1.1
-
+    -r{toxinidir}/requirements/flake8.txt
 commands = flake8
 
 [testenv:docs]
 changedir = docs
-deps = Sphinx==1.3.4
+deps =
+    -r{toxinidir}/requirements/common.txt
+    -r{toxinidir}/requirements/documentation.txt
 commands = sphinx-build -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
 
 [testenv:coverage]
 passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
 deps =
     {[base]deps}
-    coverage==4.0.3
-    coveralls==1.1
+    -r{toxinidir}/requirements/coverage.txt
 commands =
     coverage run --source bcauth,mdn,webplatformcompat,tools setup.py test
     coverage report -m
-    coveralls
+    -coveralls
```

Comment:
What does the `-` sign here do? Does that mean "run this thing, but if it dies, don't consider that an error" or something like that?